pkgs/games/warzone2100/default.nix

```diff
@@ -1,25 +1,25 @@
 { stdenv, fetchurl, bison, flex, gettext, pkgconfig, libpng
 , libtheora, openal, physfs, mesa, fribidi, fontconfig
 , freetype, qt4, glew, libogg, libvorbis, zlib, libX11
-, libXrandr, zip, unzip, which
+, libXrandr, zip, unzip, which, perl
 , withVideos ? false
 }:
 stdenv.mkDerivation rec {
   pname = "warzone2100";
-  version = "3.1.1";
+  version = "3.1.5";
   name = "${pname}-${version}";
   src = fetchurl {
     url = "mirror://sourceforge/${pname}/releases/${version}/${name}.tar.xz";
-    sha256 = "c937a2e2c7afdad00b00767636234bbec4d8b18efb008073445439d32edb76cf";
+    sha256 = "0hm49i2knvvg3wlnryv7h4m84s3qa7jfyym5yy6365sx8wzcrai1";
   };
-  sequences_src = fetchurl {
+  sequences_src = stdenv.lib.optionalString withVideos (fetchurl {
```

Comment:
You can move this to a `let` binding instead. This way, because Nix is lazy, it won't be computed (and so, downloaded) unless it's really needed, and the only place it can be needed is the `postInstall` hook, which is guarded by `optionalString`.